spec/models/partners/profile_spec.rb

```diff
@@ -167,6 +167,36 @@
     end
   end
 
+  describe "pick up email address validation" do
+    context "number of email addresses" do
+      let(:profile) { build(:partner_profile, pick_up_email: "a@b.c, a@b.c, a@b.c, a@b.c") }
```

Comment:
We are going to `uniq` these addresses - maybe the validation should `uniq` them? It's probably a mistake if they enter the same address twice.

Reply:
Thanks for your suggestion. I chose the `pick_up<n>@org.com` pattern for testing. If you prefer another one, I can change it.
Now, it will show an error for this.